ReleaseTooling/Sources/FirebaseReleaser/main.swift

```diff
@@ -88,55 +88,6 @@ struct FirebaseReleaser: ParsableCommand {
     print("Finished at: \(finishDate.dateTimeString()). " +
       "Duration: \(startDate.formattedDurationSince(finishDate))")
   }
-
-  private func updateFirebasePod(newVersions: [String: String]) {
-    let podspecFile = gitRoot.appendingPathComponent("Firebase.podspec")
-    var contents = ""
-    do {
-      contents = try String(contentsOfFile: podspecFile.path, encoding: .utf8)
-    } catch {
-      fatalError("Could not read Firebase podspec. \(error)")
-    }
-    for (pod, version) in newVersions {
-      if pod == "Firebase" {
-        // Replace version in string like s.version = '6.9.0'
-        guard let range = contents.range(of: "s.version") else {
-          fatalError("Could not find version of Firebase pod in podspec at \(podspecFile)")
-        }
-        var versionStartIndex = contents.index(range.upperBound, offsetBy: 1)
-        while contents[versionStartIndex] != "'" {
-          versionStartIndex = contents.index(versionStartIndex, offsetBy: 1)
-        }
-        var versionEndIndex = contents.index(versionStartIndex, offsetBy: 1)
-        while contents[versionEndIndex] != "'" {
-          versionEndIndex = contents.index(versionEndIndex, offsetBy: 1)
-        }
-        contents.removeSubrange(versionStartIndex ... versionEndIndex)
-        contents.insert(contentsOf: "'" + version + "'", at: versionStartIndex)
-      } else {
-        // Replace version in string like ss.dependency 'FirebaseCore', '6.3.0'
```

Comment:
I believe this else block is needed.

Reply:
@paulb777: This method was parsing `Firebase.podspec` file. If the file should be deleted (https://github.com/firebase/firebase-ios-sdk/pull/9273#discussion_r795905014), I think, this code is also not needed.
I looked through the modern version of main.swift in master and could not find any usages of `private func updateFirebasePod(newVersions: [String: String])`. Possibly it is not needed already.
So I think the file should be replaced by a master version without updateFirebasePod method.

Reply:
@a-25 I see. Thanks!